lib/smartdown_flows/inherits-someone-dies-without-will-smartdown/snippets/next_steps_wills_and_optional_inheritance_tax_links.txt

```diff
@@ -0,0 +1,19 @@
+$IF estate_over_250000 is 'yes'
```

Comment:
Hm, next steps should probably have condition evaluation (and interpolation, etc)  support added, I think thats oversight more than intent.